dart/common/StlHelpers.h

```diff
@@ -70,6 +71,21 @@ static T getVectorObjectIfAvailable(size_t index, const std::vector<T>& vec)
   return nullptr;
 }
 
+//==============================================================================
+/// static_if_else allows the compiler to choose between two different possible
+/// types at runtime based on a runtime boolean.
+template <bool B, typename T_if, typename T_else>
+struct static_if_else
```

Comment:
Yes, you are exactly right. I'll replace that right away. I was surprised when I couldn't seem to find anything similar to my ``static_if_else`` in the standard, but I guess it didn't occur to me to look for something called ``conditional``.